### PR TITLE
Index blog posts

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/giantswarm/issues/15387

The PR includes a major restructuring, to accommodate multiple indexers as CLI sub-commands.

### CODE REVIEW

**Comment by marians** [comment]
Currently testing

**Comment by marians** [comment]
@ubergesundheit @stone-z I'd appreciate a shallow glance over this. Not expecting a deep review.

File: README.md [hunk 1]
```diff
@@ -2,36 +2,56 @@
...
+
+- Documentation ([docs.giantswarm.io](https://docs.giantswarm.io/))
+  - REST API documentation ([docs.giantswarm.io/api/](https://docs.giantswarm.io/api/))
+- Blog posts ([www.giantswarm.io/blog](https://www.giantswarm.io/blog))
 
 ## App
 
 There is a helm chart in the `helm` subfolder.
 
+Note: A separate secret is required for the `HUBSPOT_API_KEY` environment variable:
```
Comment (stone-z) [inline README.md:17]: I wondered where this key was being used. TIL :+1:

**Review by stone-z — APPROVED** [review]
LGTM

**Pull request merged**